cmd/ocm-backplane/console/console.go

```diff
@@ -1113,11 +1113,8 @@ func (ce *podmanMac) runMonitorPlugin(containerName string, consoleContainerName
 }
 
 func (ce *podmanLinux) runMonitorPlugin(containerName string, consoleContainerName string, nginxConf string, pluginArgs []string) error {
-	configDirectory, err := config.GetConfigDirctory()
-	if err != nil {
-		return err
-	}
-	nginxConfPath := filepath.Join(configDirectory, nginxConf)
+	tmpDir := os.TempDir()
+	nginxConfPath := filepath.Join(tmpDir, nginxConf)
```

Comment:
Shall we append {cluster-id} to nginx.conf, otherwise this file may override when the user connects to multi-cluster consoles

Reply:
That's a good point!  Let's put a cluster_id there.